hoot-services/src/main/java/hoot/services/controllers/grail/MergeOsmFilesCommand.java

```diff
@@ -48,23 +49,31 @@
 
     private final Map<String, Object> substitutionMap = new HashMap<>();
     private final Class<?> caller;
-
-    private static final FilenameFilter filter = new FilenameFilter() {
-        @Override
-        public boolean accept(File f, String name) {
-            // We want to find only .osm files
-            return name.endsWith(".osm");
-        }
-    };
+    private final FilenameFilter filter;
 
     MergeOsmFilesCommand(String jobId, GrailParams params, String debugLevel, Class<?> caller) {
         super(jobId, params);
         logger.info("Params: " + params);
 
         this.caller = caller;
 
+        List<String> options = Collections.emptyList();
+        if (params.getAdvancedOptions() != null) {
+            options = params.getAdvancedOptions().entrySet().stream().map(ent -> {
+                return ent.getKey() + "=" + ent.getValue();
+            }).collect(Collectors.toList());
```

Comment:
since there is only 1 line in the map, the return, you can simplify this to
` options = params.getAdvancedOptions().entrySet().stream().map(ent -> ent.getKey() + "=" + ent.getValue()).collect(Collectors.toList());
`

Reply:
wanna push that change?